src/main/resources/org/jenkinsci/plugins/github_branch_source/Messages.properties

```
@@ -13,7 +13,7 @@ ForkPullRequestDiscoveryTrait.nobodyDisplayName=Nobody
 GitHubBranchFilter.DisplayName=GitHub Branch Jobs Only
 
 GitHubBuildStatusNotification.CommitStatus.Good=This commit looks good
-GitHubBuildStatusNotification.CommitStatus.Unstable=This commit has test failures
+GitHubBuildStatusNotification.CommitStatus.Unstable=This commit has failing tests or missing quality gates
```

I for one would have no idea what a _missing quality gate_ might mean. I am supposed to add a “quality gate” somewhere?

Perhaps
```suggestion
GitHubBuildStatusNotification.CommitStatus.Unstable=This commit failed tests or other validations
```
?